android/titanium/build.gradle

```diff
@@ -279,6 +279,15 @@ dependencies {
 
 	// Reference all local JAR file dependencies.
 	implementation fileTree(dir: 'lib', include: ['*.jar'])
+
+	// CameraX
+  def camerax_version = '1.1.0'
```

Comment:
Note: 1.2.0 beta 2 is available, just let's do that in a later release.

Reply:
:+1: yes, they've changed a lot in the 1.1.0 betas so we'll wait for the release in order not to change it multiple times :smile: